elevate-cpanel

```diff
@@ -10869,7 +10869,29 @@ sub run_stage_3 ($self) {
         $self->ssystem_and_die( '/usr/bin/mount', '-o', 'remount,exec', '/tmp' );
 
         INFO('Running do-release-upgrade');
-        $self->ssystem_and_die( '/usr/bin/do-release-upgrade', '-f', 'DistUpgradeViewNonInteractive' );
+        my $ret = $self->ssystem_capture_output( '/usr/bin/do-release-upgrade', '-f', 'DistUpgradeViewNonInteractive' );
+
+        if ( $ret->{status} != 0 ) {
+            my $stderr = join "\n", @{ $ret->{stderr} };
+
+            my $error_message = <<~"EOS";
+            Running do-release-upgrade has failed on your server with the
+            following error output:
```

Comment:
The do-release-upgrade script failed with the following error message:

Although do we need to name the script here? Can it be phrased "The upgrade failed with the following error message:"

Reply:
I think it is better to name the script here since the customer may want to run it manually after reviewing the logs.  Went with the top option.